websites/E/e-typing/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.7",
+  "author": {
+    "name": "Toshi",
+    "id": "548698164874575875"
+  },
+  "service": "e-typing",
+  "description": {
+    "en": "e-typing"
```

Comment:
Try getting the official description of the website, for example, from Google.
If it's in Japanese, then you can add the `jp` description and translate it to English.